src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudIT.java

```diff
@@ -231,6 +232,21 @@ public void test1WorkerCreatedFor2Executors() throws Exception {
         assertEquals(logs(), 1, planned.size());
     }
 
+    @Test(timeout = 300000)
+    public void testMultipleLabels() throws Exception {
+        // For a configuration with multiple labels, test if job label matches one of the configuration's labels
+        logOutput.reset();
+
+        ComputeEngineCloud cloud = (ComputeEngineCloud) r.jenkins.clouds.get(0);
+        InstanceConfiguration ic = instanceConfiguration(DEB_JAVA_STARTUP_SCRIPT, NUM_EXECUTORS, MULTIPLE_LABEL);
```

Comment:
This approach shifts the original issue I brought up from the provision function to the InstanceConfiguration constructor. If labelString is going to now represent a collection of labels, then it should be changed to a list type. Function args having different kinds of values depending on the use case is an example of code smell :)

Reply:
Instance Configuration gets its values from the Jenkins configuration UI, and I don't believe I can store them as a list.